config/config.go

```diff
@@ -63,6 +64,12 @@ func (c *Config) Ensure(fromCloud bool) error {
 		}
 	}
 
+	for idx := 0; idx < len(c.Modules); idx++ {
+		if err := c.Modules[idx].Validate(fmt.Sprintf("%s.%d", "modules", idx)); err != nil {
+			return err
```

Comment:
since we allow partial start up now, should we error out just because a module config is wrong?